sys-power/tlp/tlp-1.5.0.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit udev optfeature
+
+DESCRIPTION="Optimize laptop battery life"
+HOMEPAGE="https://linrunner.de/tlp/"
+SRC_URI="https://github.com/linrunner/TLP/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/TLP-${PV}"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="mirror"
+# It's uncertain if elogind/systemd is actually required, however, without the sleep
+# hooks working, which require one of them, it doesn't seem like this app is very useful.
+RDEPEND="virtual/udev
+		|| ( sys-auth/elogind sys-apps/systemd )"
+DEPEND="${RDEPEND}"
```

Comment:
Are those really DEPENDs?

Reply:
This doesn't link against these packages, just needs them for runtime so I'm only putting them as DEPEND.